.github/ISSUE_TEMPLATE/new_rule.yaml

```diff
@@ -0,0 +1,88 @@
+name: New Rule
+description: Suggestions and ideas for new rules
+title: "[New Rule] Name of rule"
+labels: ["Rule: New", "Team: TRADE"]
+assignees: []
+projects: ["elastic/1268"]
+
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: "Provide a detailed description of the activity to be detected."
+      placeholder: "Detailed description..."
+
+  - type: dropdown
+    id: target_ruleset
+    attributes:
+      label: Target Ruleset
+      description: "Select the target rulset."
+      options:
+        - apm
+        - cross-platform
+        - integrations
+        - linux
+        - macos
+        - ml
+        - network
+        - promotions
+        - threat_intel
+        - windows
+        - other
+
+  - type: dropdown
+    id: rule_type
+    attributes:
+      label: Target Rule Type
+      description: "Select the target type."
+      options:
+        - KQL
+        - ML_Job
+        - Threshold
+        - EQL
+        - Indicator_Match
+        - New_Terms
+        - ESQL
```

Comment:
Maybe we attempt to match this as close as possible to the UI to stay consistent?

- Custom (KQL or Lucene)
- Machine Learning
- Threshold
- Event Correlation (EQL)
- Indicator Match
- New Terms
- ES|QL